R/bump-version.R

```diff
@@ -95,17 +95,17 @@ get_main_branch_remote <- function(remote, repo) {
   basename(as.character(remotes$symref[remotes$ref == "HEAD"]))
 }
 
-get_main_branch_config <- function(repo) {
+  get_main_branch_config <- function(repo) {
```

Comment:
why change the indentation level?

Reply:
I'm not sure, it wasn't intentional. Could be a whitespace character artefact or some hiccup with trying to paste code from my IDE into github's online editor. ~Is there a way to get rid of that without redoing the whole PR from scratch?~

edit: I think I did it.